docs/examples/snek5000-tgv/src/snek5000_tgv/tgv.usr.f

```diff
@@ -79,14 +79,21 @@ subroutine userchk()
      &               omg(lx1*ly1*lz1*lelv,ldim)
 
       character*80 fnames(3)
+      character*128 file_name
+      logical exist
+      file_name = 'spatial_means.txt'
```

Comment:
Could you please use "spatial_mean.csv" (and try to produce a proper .csv file that can be easily read by `pd.read_csv`)?